nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -692,18 +722,18 @@ def export_model(self, model_path, calibration_path=None, onnx_path=None, input_
         calibration_config = {}
 
         for name, module in self.bound_model.named_modules():
-            if hasattr(module, 'weight_bit') or hasattr(module, 'activation_bit'):
+            if hasattr(module, 'input_bit') or hasattr(module, 'output_bit'):
                 calibration_config[name] = {}
-            if hasattr(module, 'weight_bit'):
-                calibration_config[name]['weight_bit'] = int(module.weight_bit)
-                abs_max_weight = float(module.scale * module.weight_qmax)
-                calibration_config[name]['tracked_min_input'] = -abs_max_weight
-                calibration_config[name]['tracked_max_input'] = abs_max_weight
-            if hasattr(module, 'activation_bit'):
-                calibration_config[name]['activation_bit'] = int(module.activation_bit)
-                abs_max_activation = float(module.scale * module.activation_qmax)
-                calibration_config[name]['tracked_min_activation'] = -abs_max_activation
-                calibration_config[name]['tracked_max_activation'] = abs_max_activation
+            if hasattr(module, 'input_bit'):
+                calibration_config[name]['weight_bit'] = int(module.input_bit)
```

Comment:
Why assigning `calibration_config[name]['weight_bit']` with `module.input_bit` instead of `module.weight_bit`. If 'weight_bit' is not equal to `input_bit` when setting the config, the export result will be incorrect.

Reply:
According to [here](https://github.com/microsoft/nni/blob/dddf0b9efa310443a52467854dea053854f6a134/nni/compression/pytorch/quantization_speedup/integrated_tensorrt.py#L176), `weight_bit ` is used to determine whether set input tensor's dynamic ranges or not, which I think may be not appropriate. Assigning `input_bit` to `weight_bit ` here is just to be consistent with it.

Reply:
Currently, we choose to record range of input tensor during the process of quantizing weight in the algorithm QAT. The reason why we handle it in this way is the requirement of integration with TensorRT which needs input tensor's dynamic range when setting layer precision to 8bit. So we record input dynamic range as [here](https://github.com/microsoft/nni/blob/dddf0b9efa310443a52467854dea053854f6a134/nni/compression/pytorch/quantization_speedup/integrated_tensorrt.py#L176).
And if we want to export LSQ model to TensorRT, `input dynamic range` should also be set in most situations and `input_bit` should be the same as `weight_bit`. 
However, it is still strange not to set `calibration_config[name]['weight_bit]` with `weight_bit` since we already have the value of `weight_bit` ==.

Reply:
Got it. How about changing the codes like:
```
 if hasattr(module, 'weight_bit'):
     calibration_config[name]['weight_bit'] = int(module.weight_bit)
     abs_max_input = float(module.input_scale * module.input_qmax)
     calibration_config[name]['tracked_min_input'] = -abs_max_input
     calibration_config[name]['tracked_max_input'] = abs_max_input
```

Reply:
Looks good. Completing related doc is necessary. Please refer to [overview](https://github.com/microsoft/nni/blob/master/docs/en_US/Compression/Overview.rst), [quantization](https://github.com/microsoft/nni/blob/master/docs/en_US/Compression/quantization.rst) and [Quantizer](https://github.com/microsoft/nni/blob/master/docs/en_US/Compression/Quantizer.rst). Feel free to ask me if have any questions.